stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -508,6 +431,7 @@
         <item name="streamUiThreadMessagesStart">bottom</item>
     </style>
 
+    <!--  There's no use for this o.O  -->
     <style name="StreamUi.MessageList.MediaAttachment" parent="StreamUi">
```

Comment:
MediaAttachmentViewStyle

```
context.obtainStyledAttributes(
                attrs,
                R.styleable.MediaAttachmentView,
                R.attr.streamUiMessageListMediaAttachmentStyle,
                R.style.StreamUi_MessageList_MediaAttachment
            )
```

Reply:
Android Studio couldn't find it... I'm glad that you did! Let me remove the comment then.